apps/agent/.env.production

```diff
@@ -0,0 +1 @@
+APP_ASSISTED_MIGRATION_URL=http://console.redhat.com/assisted-migration
```

Comment:
@ammont82 Would this URL go to `/migrate/wizard` or to `/migrate`

Reply:
With the new naming I think we need to go to http://console.redhat.com/migration-assessment/migrate/wizard